app/build.gradle

```diff
@@ -113,29 +113,35 @@ android {
     }
 }
 
+ext {
+    leakCanaryVersion = '1.5.1'
+    supportLibsVersion = '26.0.2'
+    firebaseVersion = '11.2.2'
+    glideVersion = '4.1.1'
+}
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     // Leak Checks and Test Cases
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    releaseImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    testImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:' + leakCanaryVersion
```

Comment:
I think "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion" will look much better.

Reply:
OK, changing it now